Profile card component

Daniel4lm opened the pull request:
## Subject

This pull request contains the solution to the Profile card component challenge on Frontend Mentor, 
as a part of my Optimum BH Internship. 

Profile card component solution provided here represents the user profile card. Card is composed of three sections, upper one with pattern image, middle one with user profile picture and lower one with user detail informations. Implemented mostly by using Flexbox CSS model and background-image CSS property(for background pattern around profile card) . 


### The solution on this branch contains the following:

- [x] Html page containing the corresponding DOM structure, including the Optimum header with anchor tags with urls(whose presense was requested by script tests).
- [x] Tailwind configuration file containing the extended theme atributes, like colors, sizes, breakpoints, etc.
- [x] Challenge starter files, incl. images and style guide
- [x] Solution design images(mobile & desktop layout)
- [x] CSS style file with tailwind directives and custom classes

Daniel4lm committed:
- Include challenge starter files

- Implement html profile card structure and tailwind css styling

Daniel4lm committed:
Modify readme.md

Daniel4lm committed:
Merge remote-tracking branch 'refs/remotes/origin/profile_card_component' into profile_card_component

Daniel4lm committed:
Add background pattern

Daniel4lm committed:
Add background pattern

Daniel4lm committed:
Minor fixes

Daniel4lm committed:
Some minor css fixes

Daniel4lm committed:
Merge branch 'optimumBA:profile_card_component' into profile_card_component

almirsarajcic commented on challenges/profile_card_component/README.md:
```diff
@@ -0,0 +1,123 @@
+@tailwind utilities;
+@layer components {
+  /* custom css class */
+  .qr-card {
```
As in the other PR, you probably want to remove this file completely.

almirsarajcic commented on challenges/profile_card_component/style-guide.md:
```diff
@@ -0,0 +1,31 @@
+# Front-end Style Guide
```
Remove this file.

almirsarajcic commented on challenges/profile_card_component/index.html:
```diff
@@ -25,39 +25,106 @@
+
+      <!-- Footer section -->
+      <footer
+        class="absolute bottom-8 rounded-md bg-white p-6 opacity-0 hover:opacity-100"
```
I can't see the credits in the screenshots.

almirsarajcic commented on challenges/profile_card_component/tailwind.config.js:
```diff
@@ -1,10 +1,56 @@
       colors: {
         'optimum-blue': '#009efc',
         'optimum-darkblue': '#0389e1',
+        primary: {
```
Nice job with the prefixes 👍
Today I learned...

almirsarajcic reviewed (changes requested):
Small issues, but good work in general 👍

Daniel4lm committed:
Update component

- Remove README.md & style-guide.ms files
- Make credits visible on the page(before was visible on mouse hover)

almirsarajcic reviewed (approved):
Nice workaround for the transparent credits issue.

Daniel4lm committed:
Fix source code link
